app/helpers/application_helper/button/generic_feature_button.rb

```diff
@@ -4,7 +4,7 @@ class ApplicationHelper::Button::GenericFeatureButton < ApplicationHelper::Butto
   def initialize(view_context, view_binding, instance_data, props)
     super
     # TODO: use #dig when ruby2.2 is no longer supported
-    @feature = props.fetch(:options, {}).fetch(:feature, nil)
+    @feature = props.fetch_path(:options, :feature)
```

Comment:
@vecerek Please remove this prevention. As we discussed, we'd like to have it in a way, that if you have a button that should check for a feature, you should always set the feature in a toolbar.